runtime/jcl/uma/se9_exports.xml

```diff
@@ -20,12 +20,8 @@
 	SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 -->
 <exports group="se9">
-	<export name="Java_java_lang_Class_getNestHostImpl">
-		<include-if condition="spec.flags.opt_valhallaNestmates" />
-	</export>
-	<export name="Java_java_lang_Class_getNestMembersImpl">
-		<include-if condition="spec.flags.opt_valhallaNestmates" />
-	</export>
+	<export name="Java_java_lang_Class_getNestHostImpl"/>
```

Comment:
This isn't going to compile/link on Windows without a definition of the function.